plugins/importexport/csv/README.md

```diff
@@ -0,0 +1,74 @@
+# CSV Import Plugin for OMP 3.3.0
+
+This application will convert a CSV file into a list of OMP publications/submissions.
```

Comment:
It's generally better to avoid adding unnecessary line breaks like this (I'm not sure if it's the case here, but in some workflows it can cause problems with translations).